mapswipe_workers/mapswipe_workers/utils/tile_grouping_functions.py

```diff
@@ -400,6 +400,9 @@ def extent_to_groups(infile, zoom, groupSize):
     # TODO: add this line once properly working
     groups_dict, overlaps_total = adjust_overlapping_groups(raw_groups_dict, zoom)
 
+    if overlaps_total == 0:
```

Comment:
hey, 
I'm not sure what happens here? Maybe my comment above was too confusing. But the line you have no has basically no effect